modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/DataEngineLayoutBuilderHandler.es.js

```diff
@@ -15,6 +15,25 @@
 export default function ({defaultLanguageId, namespace}) {
 	const form = document.getElementById(`${namespace}fm`);
 
+	const clearNameInputIfNeeded = () => {
+		const inputComponent = Liferay.component(`${namespace}name`);
+		const selectedLanguageId = document.getElementById(
+			`${namespace}languageId`
+		).value;
+		const translatedLanguages = inputComponent.get('translatedLanguages');
+
+		if (
+			!translatedLanguages.has(selectedLanguageId) &&
+			selectedLanguageId !== defaultLanguageId
+		) {
+			inputComponent.updateInput('');
+
+			const form = Liferay.Form.get(`${namespace}fm`);
```

Comment:
Similar issues can happen here. Not sure if the Form initialization is guaranteed before this code runs...

Reply:
The same here